content/en/apps/tutorials/application-settings.md

```diff
@@ -0,0 +1,184 @@
+---
+title: "CHT Application Settings"
+linkTitle: Application Settings
+weight: 1
+description: >
+  Managing CHT application settings
+relatedContent: >
+  
+---
+
+## Purpose of the Tutorial
+
+This tutorial will take you through how to manage the CHT application settings, including; setting user roles and permissions, enabling and disabling transitions, configuring contact hierarchy and configuring replication.
+
+## Brief Overview of Key Concepts
+
+The settings which control CHT apps are defined in the *app_settings.json* file, and stored in the settings doc in the database.
+
+*Permissions* are settings that control access to specific app features and functionality.
+
+*Roles* define permissions for users to perform access a group of app features and functionality.
+
+*[Replication depth](https://docs.communityhealthtoolkit.org/apps/guides/performance/replication/#depth)* is the number of levels a specific user role is able to replicate on their application.
+
+*[A transition](https://docs.communityhealthtoolkit.org/core/overview/transitions/)* is javascript code that runs when a document is changed. A transition can edit the changed doc or do anything server side code can do for that matter.
+
+## Required Resources
+
+You should have a functioning CHT instance and have medic-conf installed locally. Read [How to set up a CHT local configuration environment](setting_up_local_configuration_environment.md)
```

Comment:
would be better to use a "see also" here: https://docs.communityhealthtoolkit.org/contribute/docs/style-guide/#cross-referencing-content

Reply:
Updated.